bundles/core/src/main/java/com/adobe/cq/email/core/components/internal/models/ImageModel.java

```diff
@@ -51,10 +50,12 @@
 /**
  * Image component model class
  */
-@Model(adaptables = {Resource.class, SlingHttpServletRequest.class},
-       adapters = {Image.class, ComponentExporter.class},
-       resourceType = "core/email/components/image/v1/image",
-       defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL)
+@Model(
+    adaptables = { Resource.class, SlingHttpServletRequest.class },
+    adapters = { Image.class, ComponentExporter.class },
+    resourceType = "core/email/components/image/v1/image",
+    defaultInjectionStrategy = DefaultInjectionStrategy.OPTIONAL
+)
 public class ImageModel implements Image {
```

Comment:
Shouldn't this be called `ImageImpl` for consistency?

Reply:
This will be renamed in https://github.com/adobe/aem-core-email-components/pull/217/files#diff-1c4bd26d2bdeafafd86d644726c2cd9b366938446c5ff0196a9891ac36db2ee3